recipes/mss/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "1.2.0" %}
+
+package:
+    name: mss
+    version:  {{ version }}
+
+source:
+    fn: mss-{{ version }}.tar.gz
+    url: https://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
+    sha256: 6e15a2a7c65ce676169551660e51ced5b0718ef83c4f93104a0f6b70ed502183
+
+build:
+    number: 0
+    script: python setup.py install
+    skip: True  # [py3k]
+    entry_points:
+      - mss = mslib.msui.mss_pyui:main
+      - mswms = mslib.mswms.mswms:main
+
+requirements:
+    build:
+        - python
+        # mss import pyproj from basemap and that does not exist in 1.0.8
+        - basemap 1.0.7
+        - chameleon
+        - execnet
+        - lxml
+        - netcdf4
+        - paste
+        - pillow
+        - python-dateutil
+        - pytz
+        - requests
+        - scipy
+        - six
+        - tk 8.5.*
+    run:
+        - python
+        - basemap 1.0.7
+        - chameleon
+        - execnet
+        - lxml
+        - netcdf4
+        - paste
+        - pillow
+        - python-dateutil
+        - pytz
+        - requests
+        - scipy
+        - six
+        - tk 8.5.*
+        # FIXME: Workaround until https://github.com/conda-forge/matplotlib-feedstock/issues/36 is fixed.
```

Comment:
Now 1.2.0 has everything pinned and 1.2.1 only basemap. I tested it with defaults on linux an windows.

https://bitbucket.org/wxmetvis/mss/commits/0af86d26a4e0584dc596e1380f65cb8c0b5feaf0#chg-meta.yaml

The new checksum ;)
```

{% set version = "1.2.1" %}

package:
  name: mss
  version: {{ version }}

source:
    fn: mss-{{ version }}.tar.gz
    url: https://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
    sha256: c08e72950a1150fd2a119a7fe766f966fe5a8b73c55fa02763c7a1e5a288f915

```